crowbar_framework/app/controllers/installers_controller.rb

```diff
@@ -51,18 +51,38 @@ def start
       # the shell Process will be spawned in the background and therefore has
       # not a direct return value which we can use here
       if Crowbar::Installer.installing?
-        flash[:notice] = I18n.t(".installers.start.installation_ongoing")
+        respond_to do |format|
+          format.json do
+            head :im_used
```

Comment:
just for reference: http://httpstatusdogs.com/226-im-used

not sure if there is some more suitable status-code... 226 refers to GET and we do a POST. any suggestions?